ironfish/src/primitives/transaction.ts

```diff
@@ -76,15 +77,15 @@ export class Transaction {
       }
     })
 
+    // output description
     this.notes = Array.from({ length: _notesLength }, () => {
       // proof
       reader.seek(PROOF_LENGTH)
 
-      const note = new NoteEncrypted(reader.readBytes(ENCRYPTED_NOTE_LENGTH, true))
-      // TODO(joe): remove once rpk removed from proof and put on transaction
-      // randomized public key
-      reader.seek(32)
-      return note
+      // output note
+      const outputNote = new NoteEncrypted(reader.readBytes(ENCRYPTED_NOTE_LENGTH, true))
```

Comment:
nit: can return directly rather than assigning here since there is no subsequent `seek`

Reply:
ok, I will update it in the next cleaning up PR.